modules/oe_content_call_proposals/oe_content_call_proposals.module

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for OpenEuropa Call For Proposals content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\oe_content_call_proposals\CallForProposalsNodeWrapperInterface;
+use Drupal\oe_content_call_proposals\CallForProposalsNodeWrapper;
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter for the node form.
+ */
+function oe_content_call_proposals_form_node_form_alter(array &$form, FormStateInterface $form_state) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $form_state->getBuildInfo()['callback_object']->getEntity();
+  if ($node->bundle() !== 'oe_call_proposals') {
+    return;
+  }
+
+  $form['oe_call_proposals_deadline']['widget']['#states'] = [
+    'invisible' => [
+      'input[name="oe_call_proposals_model"]' => [
+        'value' => CallForProposalsNodeWrapperInterface::MODEL_PERMANENT,
```

Comment:
Do we need to set empty value for field `oe_call_proposals_deadline` if model is permanent as it was done for fields `oe_organisation_non_eu_org_type`, `oe_organisation_eu_org_type`,   `oe_organisation_eu_org` in `oe_content_organisation_node_presave`? Spec says nothing about it, but I assume that yes.

Reply:
This is done.